lib/system.nim

```diff
@@ -2050,20 +2050,56 @@ template unlikely*(val: bool): bool =
       unlikelyProc(val)
 
 const
+  ## TODO: in the next csources release:
```

Comment:
Sketched out what I think we should do with the compiler versions etc being exposed via the standard library/programmatically. Also need to dial the numbers back when we cut the next csources.

tl;dr: compiler version != standard library versions, also I'm completely uninterested in keep nim version compatibility, people can just edit their code.